component/component/vshn_redis.jsonnet

```diff
@@ -384,6 +384,82 @@ local composition =
                    },
                  };
 
+  local prometheusRule = comp.KubeObject('monitoring.coreos.com/v1', 'PrometheusRule') +
+                         {
+                           spec+: {
+                             forProvider+: {
+                               manifest+: {
+                                 metadata: {
+                                   name: 'redis-rules',
+                                 },
+                                 local bottomPod(query) = 'label_replace( bottomk(1, %s) * on(namespace) group_left(label_appcat_vshn_io_claim_namespace) kube_namespace_labels, "name", "$1", "namespace", "vshn-redis-(.+)-.+")' % query,
+                                 local topPod(query) = 'label_replace( topk(1, %s) * on(namespace) group_left(label_appcat_vshn_io_claim_namespace) kube_namespace_labels, "name", "$1", "namespace", "vshn-redis-(.+)-.+")' % query,
+                                 spec: {
+                                   groups: [
+                                     {
+                                       name: 'redis-memory',
```

Comment:
This looks wrong. Why is the top group called `redis-memory` but contains all kind of alerts?

Reply:
renamed it with `name: 'redis-non-slo-alerts',` that was a leftover after development, when I started with memory first